test/e2e/nix/build_test.go

```diff
@@ -0,0 +1,81 @@
+package nix_test
```

Comment:
Add a tests which assures dependencies from umbrella-child-bobfile combination are handled correctly for tasks defined in a umbrella and child.